keps/sig-apps/3998-job-success-completion-policy/kep.yaml

```diff
@@ -45,4 +45,3 @@ disable-supported: true
 
 # The following PRR answers are required at beta release
 metrics:
-  - job_succeeded_total
```

Comment:
This should state `jobs_finished_total` based on the update, not a removal of a metric, which is one of the requirement for almost every enhancement.

Reply:
It doesn't mean it has to be new metrics, but ones that are relevant for observing the state of the cluster after the feature is being used. With that I'd also add `job_sync_duration_seconds` which you mentioned in the SLI section of the enhancement.

Reply:
I did not know that. Thank you for the kindly guidance.

Reply:
done.

Reply:
+1, I think `jobs_finished_total` should still be added

Reply:
Good point. Thanks.
I added that.